support.html

```diff
@@ -57,21 +57,19 @@ <h2 id="community">Community Support</h2>
 
       <li><a href="https://lists.sourceforge.net/mailman/listinfo/vufind-admins">VuFind&reg; Admins</a> - A mailing list for discussing management of the VuFind&reg; project.</li>
 
-      <li><a href="https://lists.lehigh.edu/mailman/listinfo/vufind-unicorn-l">VuFind&reg; Unicorn Users</a> - A mailing list for SirsiDynix Unicorn users who are using VuFind&reg;.</li>
-
       <li><a href="http://vufind.slack.com/">VuFind&reg; on Slack</a> - Email info@vufind.org for an invitation to the Slack if you cannot sign up directly.</li>
 
-      <li><strong>VuFind&reg; on IRC</strong> - For real-time discussion of VuFind&reg;, try joining the vufind channel at <a href="http://webchat.freenode.net/?channels=vufind">freenode</a>.</li>
```

Comment:
I never check IRC, and I think Slack has replaced it, so I'm removing it. Also adding in GitHub Discussions since that's a new option that we might as well mention here.